egg_updater.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+from io import StringIO
+
+from setuptools.dist import Distribution
+
+
+def IS_PRODUCTION():
+    return any(
```

Comment:
```suggestion
    return not any(
```

`.egg-link` is present only if the package is installed in development mode

Reply:
I don't have .egg-link present.

Reply:
I guess we don't need another check for this. since, egg_updater is at root level. it won't get included anyway in production version.

Reply:
Interesting okay let's just ditch this check then.